### DISCUSSION TITLE
Measurement composition

### DISCUSSION THREAD

**Discussion by ghost** [OpenGammaProject/Open-Gamma-Detector#22] [General]
Hi,

I really like your project and I bought this Detector PCB and a SiPM too. The question is could you give me some further information of your measurement compositions? I mean about gain, over voltage, scintillator size and etc. Perhaps, did you take measurements with CO60?  
I would really appreciate if you reply for my questions.

Best regards,
Peter

**Comment by NuclearPhoenixx** [comment]
Hi,

I'm using a 18 x 30 mm NaI(Tl) crystal with a 6 mm MicroFC SiPM. Bias Voltage is about 29.6 Volts. Gain is set to about 40, I'm not 100% sure about the exact figure here.

I don't have Co-60 here, so I didn't have the chance to measure it yet. Na-22 is the one with the highest energies I have tested yet, I think.

**Comment by NuclearPhoenixx** [comment]
Closing this for now since it looks like I answered your question.
